iset.mm

```diff
@@ -81701,6 +81799,28 @@ A Cauchy sequence (as defined here, which has a rate convergence built
     $( $j restatement 'ax-caucvg' of 'axcaucvg'; $)
   $}
 
+  ${
+    $d A x y z $.
+    $( An inhabited, bounded-above, located set of reals has a supremum.
+
+       Locatedness here means that given ` x < y ` , either there is an element
+       of ` A ` greater than ` x ` , or ` y ` is an upper bound.
+
+       Although this and ~ ax-caucvg are both completeness properties,
+       countable choice would probably be needed to derive this from
+       ~ ax-caucvg .
```

Comment:
But this Dedekind completeness implies Cauchy completeness in IZF (even CZF), correct ? (else, I'd have to correct https://us.metamath.org/ileuni/ax-ddkcomp.html)

Reply:
I think so, although most of the sources don't exactly address this expressed in quite this way, so I've been a little bit weasel wording statements of this sort unless we prove something.